ubuntu_install_requirements.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+set -ex
+
+source lib/logging.sh
+
+# sudo apt install -y libselinux-utils
+# if selinuxenabled ; then
+#     sudo setenforce permissive
+#     sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+# fi
+
+# Update to latest packages first
+sudo apt -y update
+
+# Install EPEL required by some packages
+# if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
+#     if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+#         sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/epel-release-7-11.noarch.rpm
+#     else
+#         sudo yum -y install epel-release --enablerepo=extras
+#     fi
+# fi
+
+# Work around a conflict with a newer zeromq from epel
+# if ! grep -q zeromq /etc/yum.repos.d/epel.repo; then
+#   sudo sed -i '/enabled=1/a exclude=zeromq*' /etc/yum.repos.d/epel.repo
+# fi
+
+# Install required packages
+
+sudo apt -y install \
+  crudini \
+  curl \
+  dnsmasq \
+  figlet \
+  zlib1g-dev \
+  libssl1.0-dev \
+  nmap \
+  patch \
+  psmisc \
+  python-pip \
+  wget
+
+# Check if 'ifconfig' is available!
+if [[ ! $(ifconfig) ]]; then
+  echo "Cannot run ifconfig..."
+  echo "Install net-tools package"
+  sudo apt -y install net-tools
+fi
+
+
+# Install pyenv
+if [[ $PATH != *pyenv* ]]; then
+  export PYENV_ROOT="$HOME/.pyenv"
+  export PATH="$PYENV_ROOT/bin:$PATH"
+  if command -v pyenv 1>/dev/null 2>&1; then
+    eval "$(pyenv init -)"
+  fi
+fi
+
+if [[  $(cat ~/.bashrc) != *PYENV_ROOT* ]]; then
+  if ! [ -d "$HOME/.pyenv" ] ; then
+     git clone git://github.com/yyuu/pyenv.git ~/.pyenv
+  fi
+  echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
```

Comment:
This cannot run twice or it will duplicate entries in the bashrc. I would rather not add anything in bashrc but make a small file to source when you want to work with pyenv

Reply:
This will not run twice as there is a check if `PYENV_ROOT` is inside .bashrc. At the moment, let keep it simple as adding something to bashrc file. We can improve it in the future.